src/Xrefcheck/Verify.hs

```diff
@@ -595,13 +602,15 @@ verifyReference
         checkAnchorReferenceAmbiguity file fileAnchors origAnchor
 
     checkAnchorExists givenAnchors anchor =
-      case find ((== anchor) . aName) givenAnchors of
+      case find (anchorNameEq anchor . aName) givenAnchors of
         Just _ -> pass
         Nothing ->
-          let isSimilar = (>= scAnchorSimilarityThreshold cScanners)
-              similarAnchors =
-                filter (isSimilar . realToFrac . damerauLevenshteinNorm anchor . aName)
-                givenAnchors
+          let isSimilar      = (>= scAnchorSimilarityThreshold cScanners)
+              distance       = on damerauLevenshteinNorm toCaseFold
+              similarAnchors = flip filter givenAnchors $ isSimilar
```

Comment:
Could you please move `isSimilar` to the next line so that it looked uniformly with other functions in the pipeline?